assets/post.css

```diff
@@ -31,146 +34,165 @@ h1, h2, h3, h4, h5, h6,
 p, blockquote, pre,
 ul, ol, dl, figure, hr,
 .highlight {
-  margin-bottom: 15px; }
-
-  .tile {
-      position: relative;
-      margin-left: 0;
-      margin-right: 0;
-      display: table;
-  }
+  margin-bottom: 15px;
+}
+.tile {
+  position: relative;
+  margin-left: 0;
+  margin-right: 0;
+  display: table;
+}
+.tile-content {
+  height: inherit;
+  width: inherit;
+  display: table-cell;
+  vertical-align: middle;
+  text-align: center;
+  padding-left: 10%;
+  padding-right: 10%;
+}
+#title {
+  float: right;
+  background-color:#4FBBA9;
+}
   .tile-content {
-      height: inherit;
-      width: inherit;
-      display: table-cell;
-      vertical-align: middle;
-      text-align: center;
-      padding-left: 10%;
-      padding-right: 10%;
-      }
-  #title {
-      float: right;
-      background-color:#4FBBA9;}
-      .tile-content {
-        background-image:url(assets/CH1.png)
-         ;
-      }
-  #ch1 {
-      float: right;}
-
-  #intro {
-      float: left;
-      background-color: #393939;
-      color: white;}
-  #soon {
-      background-color:rgba(79, 187, 169, 0.7);
-      float: left;}
+    background-image:url(assets/CH1.png);
+    }
```

Comment:
@shaqsingh right now this isn't doing anything... wanted to flag to see if this needs to be corrected or safe to remove